src/Adapter/Driver/IbmDb2/Statement.php

```diff
@@ -238,4 +247,31 @@ public function execute($parameters = null)
         $result = $this->driver->createResult($this->resource);
         return $result;
     }
+
+    /**
+     * Creates and returns a callable error handler that raises exceptions.
+     *
+     * Only raises exceptions for errors that are within the error_reporting mask.
+     *
+     * @return callable
+     */
+    private function createErrorHandler() : callable
```

Comment:
zend-db still use [php 5.5/5.6](https://github.com/zendframework/zend-db/blob/8dfeabc664dd45c5f1753fbb508db70d0e605bd4/composer.json#L16) so the return type won't work